sdk/src/cose_validator.rs

```diff
@@ -887,7 +887,7 @@ fn validate_with_cert(
     let pk_der = pk.raw;
 
     if validator.validate(sig, data, pk_der)? {
-        Ok(extract_subject_from_cert(&signcert)?)
+        extract_subject_from_cert(&signcert).or_else(|_| Ok("".to_string()))
```

Comment:
This might read better as `Ok(extract_subject_from_cert(&signcert).unwrap_or_default())`.

`String::default()` is the empty string.

Reply:
implemented